<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameEditorForm</class>
 <widget class="QDialog" name="NameEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Titles</string>
       </property>
       <property name="buddy">
        <cstring>titles</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titles"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="givenNamesLabel">
       <property name="text">
        <string>Given names</string>
       </property>
       <property name="buddy">
        <cstring>givenNames</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="givenNames"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="surnameLabel">
       <property name="text">
        <string>Surname</string>
       </property>
       <property name="buddy">
        <cstring>surname</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <widget class="QLineEdit" name="prefix">
         <property name="placeholderText">
          <string>Prefix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surname"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="originLabel">
       <property name="text">
        <string>Origin</string>
       </property>
       <property name="buddy">
        <cstring>origin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="origin">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="noteLabel">
       <property name="text">
        <string>Note</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout2_2" stretch="0,0">
       <item>
        <widget class="KRichTextEdit" name="noteEdit"/>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignTop">
        <widget class="QPushButton" name="noteEditButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="edit-entry"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>QTextEdit</extends>
   <header>KRichTextEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>NameEditorForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>NameEditorForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
